coverage.txt

```diff
@@ -0,0 +1,3325 @@
+mode: atomic
```

Comment:
Is this required to be added?

Reply:
No this was committed by mistake will removed